### ISSUE TITLE
Broken test cases in the SDLMenuManagerSpec

### ISSUE THREAD

**Issue by NicoleYarroch** [smartdevicelink/sdl_ios#1391]
### Bug Report
Two notifications are sent with the wrong name in the `SDLMenuManagerSpec` and fail when testing.

```
caught "NSInvalidArgumentException", "-[SDLSetDisplayLayoutResponse hmiCapabilities]: unrecognized selector sent to instance 0x7f983c9099c0"
```

The following code:
```
describe(@"Notificaiton Responses", ^{
    it(@"should set display capabilities when SDLDidReceiveSetDisplayLayoutResponse is received", ^{
        testDisplayCapabilities = [[SDLDisplayCapabilities alloc] init];

        testSetDisplayLayoutResponse = [[SDLSetDisplayLayoutResponse alloc] init];
        testSetDisplayLayoutResponse.success = @YES;
        testSetDisplayLayoutResponse.displayCapabilities = testDisplayCapabilities;

        SDLRPCResponseNotification *notification = [[SDLRPCResponseNotification alloc] initWithName:SDLDidReceiveRegisterAppInterfaceResponse object:self rpcResponse:testSetDisplayLayoutResponse];
        [[NSNotificationCenter defaultCenter] postNotification:notification];

        expect(testManager.displayCapabilities).to(equal(testDisplayCapabilities));
    });

    it(@"should set display capabilities when SDLDidReceiveRegisterAppInterfaceResponse is received", ^{
        testDisplayCapabilities = [[SDLDisplayCapabilities alloc] init];

        testRegisterAppInterfaceResponse = [[SDLRegisterAppInterfaceResponse alloc] init];
        testRegisterAppInterfaceResponse.success = @YES;
        testRegisterAppInterfaceResponse.displayCapabilities = testDisplayCapabilities;

        SDLRPCResponseNotification *notification = [[SDLRPCResponseNotification alloc] initWithName:SDLDidReceiveSetDisplayLayoutResponse object:self rpcResponse:testRegisterAppInterfaceResponse];
        [[NSNotificationCenter defaultCenter] postNotification:notification];

        expect(testManager.displayCapabilities).to(equal(testDisplayCapabilities));
    });
});
```

Should be:
```
describe(@"Notification Responses", ^{
    it(@"should set display capabilities when SDLDidReceiveSetDisplayLayoutResponse is received", ^{
        testDisplayCapabilities = [[SDLDisplayCapabilities alloc] init];

        testSetDisplayLayoutResponse = [[SDLSetDisplayLayoutResponse alloc] init];
        testSetDisplayLayoutResponse.success = @YES;
        testSetDisplayLayoutResponse.displayCapabilities = testDisplayCapabilities;

        SDLRPCResponseNotification *notification = [[SDLRPCResponseNotification alloc] initWithName:SDLDidReceiveSetDisplayLayoutResponse object:self rpcResponse:testSetDisplayLayoutResponse];
        [[NSNotificationCenter defaultCenter] postNotification:notification];

        expect(testManager.displayCapabilities).to(equal(testDisplayCapabilities));
    });

    it(@"should set display capabilities when SDLDidReceiveRegisterAppInterfaceResponse is received", ^{
        testDisplayCapabilities = [[SDLDisplayCapabilities alloc] init];

        testRegisterAppInterfaceResponse = [[SDLRegisterAppInterfaceResponse alloc] init];
        testRegisterAppInterfaceResponse.success = @YES;
        testRegisterAppInterfaceResponse.displayCapabilities = testDisplayCapabilities;

        SDLRPCResponseNotification *notification = [[SDLRPCResponseNotification alloc] initWithName:SDLDidReceiveRegisterAppInterfaceResponse object:self rpcResponse:testRegisterAppInterfaceResponse];
        [[NSNotificationCenter defaultCenter] postNotification:notification];

        expect(testManager.displayCapabilities).to(equal(testDisplayCapabilities));
    });
});
```

##### Reproduction Steps
1. Run all the test cases

##### Expected Behavior
All test cases pass.

##### Observed Behavior
Sometimes the above test cases fail.

##### OS & Version Information
* iOS Version: n/a
* SDL iOS Version: 6.3
* Testing Against: n/a

**Comment by NicoleYarroch** [comment]
This will be fixed in PR #1347.

**Comment by NicoleYarroch** [comment]
This was fixed in PR #1347
